DSL/Ruuter.private/PUT/admin/participants.yml

```diff
@@ -0,0 +1,72 @@
+# TODO uncomment lines 2-5, use actual cookie from header for authentication.
+# check_for_cookie:
+#   switch:
+#     - condition: ${incoming.headers == null || incoming.headers.cookie == null}
+#       next: return_unauthorized
+
+authenticate:
+  template: templates/permissions/admin
+  requestType: post
+  headers:
+    cookie: cookie #${incoming.headers.cookie}
```

Comment:
Note: To be replaced with tim jwt cookie (In Ruuter Guards)

Reply:
Ok